flojoy/plotly_utils.py

```diff
@@ -63,6 +63,10 @@ def data_container_to_plotly(data: DataContainer) -> dict[str, Any]:
             )
         case "plotly":
             fig = cast(go.Figure, data.fig)
+        case "bytes":
+            fig = go.Figure()
+        case "text_blob":
+            fig = go.Figure()
```

Comment:
Can't we utilize the Table plot for visualizing string values?

Reply:
Yes, we can, but we had a discussion in slack about creating a new frontend component for displaying text values.
I'll check if this is already implemented to return the best values here.
@smahmed776 do you have any suggestions on it?